qos/evm/store.go

```diff
@@ -0,0 +1,66 @@
+package evm
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/buildwithgrove/path/relayer"
+)
+
+// EndpointStore provides the endpoint selection capability required
+// by the relayer package for handling a service request.
+var _ relayer.EndpointSelector = &EndpointStore{}
+
+type EndpointStoreConfig struct {
```

Comment:
Yes, I have updated the comment with the following:

```
EndpointStoreConfig captures the settings of the EndpointStore that can be modified.
This will potentially allow using the `EndpointStore` as part of QoS for multiple EVM-based blockchains
which may have different desired QoS properties.
e.g. different blockchains QoS instances could have different levels tolerance for deviation from
the current block height.

```